integration_tests/tests/flowmachine_tests/test_available_dates.py

```diff
@@ -0,0 +1,91 @@
+import json
+import pytest
+
+from flowmachine.core.available_dates import AvailableDates
+
+
+@pytest.mark.parametrize(
```

Comment:
Should you not test None?

Reply:
I'm doing that in the test below (`test_available_dates_without_explicit_event_types`) because I didn't want the list of expected results to become crazy long. Do you think this is sufficient?

Reply:
Yeah sorry didn't see!